api/src/main/java/org/apache/cloudstack/api/command/user/offering/ListDiskOfferingsCmd.java

```diff
@@ -44,6 +45,12 @@ public class ListDiskOfferingsCmd extends BaseListDomainResourcesCmd {
     @Parameter(name = ApiConstants.NAME, type = CommandType.STRING, description = "name of the disk offering")
     private String diskOfferingName;
 
+    @Parameter(name = ApiConstants.ACCOUNT, type = CommandType.STRING, description = "list resources by account. Must be used with the domainId parameter.")
```

Comment:
@RodrigoDLopez 
you can use `BaseListProjectAndAccountResourcesCmd` instead of `BaseListDomainResourcesCmd`
then these are not needed.